frameworks/hdfs/src/main/dist/core-site.xml

```diff
@@ -517,34 +519,6 @@
         <value>{{TFILE_FS_INPUT_BUFFER_SIZE}}</value>
     </property>
     <property>
-        <name>hadoop.http.authentication.type</name>
-        <value>{{HADOOP_HTTP_AUTHENTICATION_TYPE}}</value>
-    </property>
-    <property>
-        <name>hadoop.http.authentication.token.validity</name>
-        <value>{{HADOOP_HTTP_AUTHENTICATION_TOKEN_VALIDITY}}</value>
-    </property>
-    <property>
-        <name>hadoop.http.authentication.signature.secret.file</name>
-        <value>{{HADOOP_HTTP_AUTHENTICATION_SIGNATURE_SECRET_FILE}}</value>
-    </property>
-    <property>
-        <name>hadoop.http.authentication.cookie.domain</name>
-        <value>{{HADOOP_HTTP_AUTHENTICATION_COOKIE_DOMAIN}}</value>
-    </property>
-    <property>
-        <name>hadoop.http.authentication.simple.anonymous.allowed</name>
-        <value>{{HADOOP_HTTP_AUTHENTICATION_SIMPLE_ANONYMOUS_ALLOWED}}</value>
-    </property>
-    <property>
-        <name>hadoop.http.authentication.kerberos.principal</name>
-        <value>{{HADOOP_HTTP_AUTHENTICATION_KERBEROS_PRINCIPAL}}</value>
-    </property>
-    <property>
-        <name>hadoop.http.authentication.kerberos.keytab</name>
```

Comment:
Is this the path/url to download the keytab? or where the keytab gets placed after downloaded from secret store?

Reply:
It's where the keytab gets downloaded to.